### PR TITLE
WE-6523: Reverses the direction of search icons and updates their styles.

### PR DESCRIPTION
**Pull request by marineb**
https://jira.wnyc.org/browse/WE-6523

### CODE REVIEW

File: app/styles/_icons.scss [hunk 1]
```diff
@@ -252,10 +252,15 @@ i.fa-circle-o-notch.fa-spin {
...
-  height: 21px;
+  width: 20px;
+  height: 23px;  
 }
 
+.searchbox--slim .icon--search {
+  width: 14px;
+  height: 15px;
+} 
+
```
Comment (marineb) [inline app/styles/_icons.scss]: Not really proud of that part but couldn't think of something better to do.

Comment (noslouch) [inline app/styles/_icons.scss]: is `.icon--search` used anywhere else? if not, you might be able to get away with just updating the base class.

Comment (marineb) [inline app/styles/_icons.scss]: Yes, it's used elsewhere, on the /shows page. That's why I had to do that.

Comment (noslouch) [inline app/styles/_icons.scss]: This is a good opportunity to refactor the search box css to use functional-semantic class names (`.site-nav-searchbox`) with design-semantic mixins (`@mixin gray-searchbox`), in the same manner that we've started to refactor the buttons file. let's talk a bit about it.

Comment (marineb) [inline app/styles/_icons.scss]: Cool. Maybe we can talk tomorrow after the standup. I don't think I'll have time to work on it today.

**Comment by marineb** [comment]
@noslouch added search mixins as discussed. Let me know what you think!

**Pull request merged**